lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1158,6 +1159,19 @@ LogicalResult FIRRTLLowering::visitExpr(ValidIfPrimOp op) {
 // Statements
 //===----------------------------------------------------------------------===//
 
+LogicalResult FIRRTLLowering::visitStmt(InvalidOp op) {
+  auto dest = getLoweredValue(op.operand());
+
+  if (!dest)
+    return failure();
+
+  auto zero = builder->create<rtl::ConstantOp>(
```

Comment:
ConstantOp has a builder that takes a constant value and a type, I think you can pass:
`(0, dest.getType().cast<IntegerType>())` to simplify this